rollcall/values.yaml

```diff
@@ -61,4 +61,10 @@ appConfig:
 podSecurityContext:
   runAsGroup: 9999
   runAsNonRoot: true
-  runAsUser: 9999
\ No newline at end of file
+  runAsUser: 9999
+
+# choose rbac roles to enable
+rbac:
+  create:
+    # for vault kubernetes integration
+    tokenReview: true
```

Comment:
I would default this to false

Reply:
thanks 👍 yeah I copy and pasted, didn't pay much attention. will update.